spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -311,6 +311,18 @@
     and_I_cannot_publish_or_schedule_the_step_by_step
   end
 
+  scenario "Fixing issues identified in the inset prompt, clears the inset prompt" do
+    given_there_is_a_step_by_step_page_with_steps
+    and_it_has_been_edited_since_its_links_were_last_checked
+    then_I_should_see_an_inset_prompt
+    and_the_prompt_should_link_to_the_check_links_page
+    but_when_I_check_for_broken_links
+    then_there_should_be_no_inset_prompt
+    and_when_I_schedule_the_step_by_step
+    then_the_step_by_step_is_scheduled
+    and_there_should_continue_to_be_no_inset_prompt
```

Comment:
Perhaps this should be added to one of the scheduling tests that check that a step-by-step can't be edited.